tests/Odbo/LimitSqlInjectionTest.php

```
@@ -0,0 +1,106 @@
+<?php
+
+namespace tests\Odbo;
+
+use stdClass;
+
+/**
+ * @covers \ODBO::get
+ */
+class LimitSqlInjectionTest extends OdboTestCase
+{
+	public function testLimitIsRejectedWhenNotNumeric()
+	{
+		$this->testModel->get([
+			'start' => 0,
+			'rows' => 'a',
+		]);
+
+		$this->assertNotError($this->testModel);
```

Assert not error? Should this fail silently or do we want it to be rejected?